jenkins-pipelines/perf-regression-latency-250gb-with-nemesis-with-hw-checks.jenkinsfile

```diff
@@ -0,0 +1,11 @@
+#!groovy
```

Comment:
@aleksbykov I don't think it should be a new pipeline, we should add it to all official pipelines of performance regressions we are switching to.